STDatabase/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/database/controllers/AppSettingsController.kt

```diff
@@ -116,5 +127,13 @@ class AppSettingsController(private val realmProvider: RealmProvider) {
     suspend fun removeData() = runThrowingRealm {
         realm.write { deleteAll() }
     }
+
+    @Throws(RealmException::class, CancellationException::class)
+    suspend fun setEmailAndToken(email: String, token: String) = runThrowingRealm {
```

Comment:
```suggestion
    suspend fun setEmailToken(email: String, token: String) = runThrowingRealm {
```

?

Since you set the `EmailToken` in DB ?